Project.toml

```diff
@@ -8,6 +8,7 @@ Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 NearestNeighbors = "b8a86587-4115-5ab1-83bc-aa920d37bbce"
 Neighborhood = "645ca80c-8b79-4109-87ea-e1f58159d116"
+Peaks = "18e31ff7-3703-566c-8e60-38913d67486b"
```

Comment:
Why is it necessary to use `Peaks` here?

Reply:
No, we need to look for the first local minimum (in `garcia_embedding_cycle!`).

Reply:
Yes, but the method you use just compares `x[i-1] > x[i] && x[i] < x[i+1]` correct? No reason to use an external package for such a simple function. Are you using a more complicated method that does e.g. gaussian smoothening first or something else?

Reply:
No, but there might be ideas to restrict the minima-search further in the future, i.e. min. peak distances or something like that. Also a smoothing first would be desirable I guess in order to avoid picking every mini-up-down. So far the n-statistics I had a look on were kind of smooth, so there hasn't been the need for it right now. If you think it would be way to much overhead then we do it your way. But this would be trickier to extend, if one wishes. I don't care so much to be honest, because this method does not give the promised results anyway.

Reply:
I still don't understand from your answer what does `Peaks` do. Does it do the simple check of finding the minimum `i` that satisfies `x[i-1] > x[i] && x[i] < x[i+1]` or not? Can you justify why we need it?

Reply:
> [...] so there hasn't been the need for it right now. If you think it would be way to much overhead then we do it your way. 

> I don't care so much to be honest, because this method does not give the promised results anyway.

Again, if one would like to improove this algorithm further it would be handy having used `Peaks` in the first place. Right now, as far as I can see, it takes the 1st minimum which satisfies `x[i-1] > x[i] && x[i] < x[i+1]`

Reply:
> Again, if one would like to improove this algorithm further it would be handy having used Peaks in the first place. 

I don't agree with this statement, because I don't believe that `Peaks` will be improved with more functionality in the near future. 

> Right now, as far as I can see, it takes the 1st minimum which satisfies `x[i-1] > x[i] && x[i] < x[i+1]`

Then we should drop the `Peaks` dependency and write the 3-lines of code function that does this operation. Pre-1.0 dependencies are a very heavy price to pay and should be avoided when it is easy to do so...

Reply:
> Then we should drop the Peaks dependency and write the 3-lines of code function that does this operation. Pre-1.0 dependencies are a very heavy price to pay and should be avoided when it is easy to do so...

alright.